akka-actor/src/main/scala/akka/pattern/CircuitBreaker.scala

```diff
@@ -563,10 +566,10 @@ class CircuitBreaker(
 
         f.onComplete {
           case s: Success[_] ⇒
-            notifyCallSuccessListeners(System.nanoTime() - start)
+            notifyCallSuccessListeners(start)
```

Comment:
@patriknw I am not sure if you are right. I am passing `start` time of call to `notifyCall*` methods in order to call System.nanoTime() only when there is any listener. Thats why all of them are `start`.